src/openassetio-core/src/pluginSystem/CppPluginSystem.cpp

```diff
@@ -158,7 +164,7 @@ const CppPluginSystem::PathAndPlugin& CppPluginSystem::plugin(const Identifier&
 CppPluginSystem::MaybeIdentifierAndPlugin CppPluginSystem::maybeLoadPlugin(
     const std::filesystem::path& filePath) {
   // Check the proposed path is actually a file.
-  if (!std::filesystem::is_regular_file(filePath)) {
+  if (!is_regular_file(filePath)) {
```

Comment:
This (and similar) changes dropping the `std::filesystem::` namespace puzzled me, as I couldn't see a `using namespace`, and it's used in the signature.

I'm supposing that because the argument is of type `std::filesystem::path` that the compiler can resolve it from all possible names of `is_regular_file`?

It was a bit of a surprise change.

Reply:
This is a result of CLion's linter (rather than Clang-Tidy) recommending the use of ADL.  I guess its a personal preference, but I liked the conciseness.

In general my process for these changes is to go through the errors/warnings/hints highlighted by CLion inline in the source file, which are largely Clang-Tidy issues, but occasionally CLion's own hints, which I've ad-hoc also applied.

Do you think I should avoid such ADL refactors?

Reply:
It came as a surprise, but if all compilers can handle it, I don't disagree with the concise nature of it.
Maybe I don't look at enough variety of C++ code these days.

If it doesn't conflict with other linters (not everyone uses CLion) and compilers, then I don't see a reason to change.